main.py

```diff
@@ -40,13 +39,7 @@ async def poker(ctx):
 
 @bot.event
 async def on_message(message):
-    await bot.process_commands(message)
-    if message.content.replace("$", "") not in command_list:
-        similar_commands = get_close_matches(message.content, command_list)
-        try:
-            await message.channel.send(f"Syntax: ${similar_commands[0]}")
-        except:
-            pass
-
+  await bot.process_commands(message)
+  await guess_command(bot, message)
```

Comment:
WHAT IS THIS ?

Reply:
CHECK THIS OUT  LOL
https://github.com/CPE35-Developer/discord-bot/blob/12ca84e9636dd473acffed1535d72f3f9175847d/src/utils/utils.py